transformer_engine/common/recipe/__init__.py

```diff
@@ -106,6 +106,20 @@ def scaling_factor_compute(amax: Tensor,
                 GPU maintains local amaxes and scaling factors. To ensure results are
                 numerically identical across checkpointing boundaries in this case, all
                 ranks must checkpoint in order to store the local tensors.
+    fp8_dpa: bool, default = `False`
```

Comment:
Please mark this and fp8_mha options as BETA functionality

Reply:
Done. Let me know if the language needs tweaking.